mkdocs.yml

```diff
@@ -58,6 +59,9 @@ markdown_extensions:
 
 plugins:
 - search
+- gen-files:
+    scripts:
+    - scripts/gen_doc_stubs.py
 - section-index
```

Comment:
Is this still needed?

Reply:
Yes, it's 2 different things. Clickable sections in the nav. Build the site and check it out if you haven't lately :sweat_smile:

Reply:
Ah, I'm dumb, I checked the published docs, which are not up-to-date. Thanks :slightly_smiling_face: